_includes/masthead-community.html

```diff
@@ -1,9 +1,69 @@
+<section class="governance">
+  <div class="wrap">
+    <div class="inner-box">
+      <div class="heading-box">
+        <h2 id="governance">Governance</h2>
+				<p>{{site.data.governance.introText}}</p>
+      </div>
+      <div class="community">
+        <div class="scala-items-list">
+          <div class="items-menu">
+            {% for scalaItem in site.data.governance.cards %}
+              {% assign loopIndexMod = forloop.index | minus: 1 | modulo: 3 %}
+
+              {% if loopIndexMod == 0 %}
+                {% assign codeSnippets = '' | split: ',' %}
+                <div class="wrap">
+              {% endif %}
+
+              <!-- Show the below content only if the scalaItem.expandable either doesn't exist or is false -->
+              {% unless scalaItem.dontExpand %}
+              {% capture box_content %}
+							<div class="wrap">
+								<ul class="vertical-buttons-list">
+									{% for link in scalaItem.links %}
+									{% if link.text %}
+									<li><p>{{ link.text }}</p></li>
+									{% else %}
+									<li><a href="{{ link.url }}" class="button button-expand" target="_blank" rel="none">{{ link.name }}</a></li>
+									{% endif %}
+									{% endfor %}
+								</ul>
+							</div>
+              {% endcapture %}
+              {% assign codeSnippets = codeSnippets | push: box_content %}
+              {% endunless %}
+              <div class="scala-item scala-item__governance">
+								<div class="top">
+									<h4>{{scalaItem.title}}</h4>
+									<p>{{scalaItem.description}}</p>
+								</div>
+                {% unless scalaItem.dontExpand %}
+                <div class="button button-more">{{scalaItem.expandText}}</div>
+                {% endunless %}
+              </div>
+              {% if loopIndexMod == 2 or forloop.last %}
+                </div>
+                <div class="items-content">
+                  {% for snippet in codeSnippets %}
```

Comment:
Do we have code snippets?

Reply:
that's a leftover from copy-paste, but its the contents of all the boxes